src/CommunityToolkit.Maui.Analyzers.UnitTests/Verifiers/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -15,6 +15,8 @@ public Test(params Type[] assembliesUnderTest)
 		{
 #if NET8_0
 			ReferenceAssemblies = ReferenceAssemblies.Net.Net80;
+#elif NET9_0
+			ReferenceAssemblies = ReferenceAssemblies.Net.Net90;
 #else
```

Comment:
We can delete the `#if NET8_0` block here since we're never going back to it

I designed the code like this so that we wouldn't forget to update the `ReferenceAssemblies` in the analyzer unit tests. 

```suggestion
#if NET9_0
			ReferenceAssemblies = Microsoft.CodeAnalysis.Testing.ReferenceAssemblies.Net.Net90;
#else
```